src/Aquifer.Data/Entities/BibleBookContentEntity.cs

```diff
@@ -22,4 +27,38 @@ public class BibleBookContentEntity : IHasUpdatedTimestamp
 
     [SqlDefaultValue("getutcdate()")]
     public DateTime Updated { get; set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// POCO for deserializing the AudioUrls column.
+    /// </summary>
+    public sealed class AudioUrlsData
+    {
+        public required IReadOnlyList<Chapter> Chapters { get; set; }
+
+        public sealed class Chapter
+        {
+            /// <summary>
+            /// The chapter number is a JSON string type but is currently only populated with integer values (this might change in the future).
+            /// </summary>
+            public required string Number { get; set; }
+            public required AudioUrl? Webm { get; set; }
+            public required AudioUrl? Mp3 { get; set; }
```

Comment:
It was clear to me from the JSON data if these are nullable or not.  I assumed that they can be `null` just in case.